src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/annotation/suppresswarnings/InputSuppressWarningsCompactNonConstant1.java

```diff
@@ -0,0 +1,92 @@
+//non-compiled with eclipse: The value for annotation attribute must be a constant expression
+package com.puppycrawl.tools.checkstyle.checks.annotation.suppresswarnings;
+import java.lang.annotation.Documented;
+
+@SuppressWarnings({"unchecked", "unused"})
+public class InputSuppressWarningsCompactNonConstant1
```

Comment:
We have to have either top comment or `Config` comment

Reply:
btw why CI does not fail with violation? no top and no config

Reply:
Here is root of problem:
```
     <module name="Regexp">
       <property name="id" value="ConfigCommentOnTopInputs"/>
-      <property name="format" value="(/* Config|\A/*)"/>
+      <property name="format" value="( \* Config|/\* Config|\A/\*)"/>

```

@shashwatj07 , please create new PR to update this Regexp and place in suppression all missed violations.
after merge of it, all other PRs should be rebased.

Reply:
> @shashwatj07 , please create new PR to update this Regexp and place in suppression all missed violations.
after merge of it, all other PRs should be rebased.

PR opened https://github.com/checkstyle/checkstyle/pull/10154

Reply:
merged, please rebase and check

Reply:
added another commit for #10115